config/locales/en.yml

```diff
@@ -668,7 +668,7 @@ en:
         client_messages: Messages
         client_notes: Notes
         client_profile: Client Profile
-        client_security: Security
+        client_security_html: "<div class='security-wrapper'>Security %{fraud_icon}</div>"
```

Comment:
i wonder if `tab_navigation_link` could be refactored into a partial instead of a helper? i haven't thought about that too hard and it's a little out of scope so i'd say this is fine unless you feel particularly enthused about pursuing a refactor